### PR TITLE
Add ZWave binding

### PR DESCRIPTION
**Pull request by cdjackson**
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: launch/app/pom.xml [hunk 1]
```diff
@@ -32,6 +32,11 @@
       <type>pom</type>
       <scope>runtime</scope>
     </dependency>
+     <dependency>
```
Comment (kaikreuzer) [inline launch/app/pom.xml]: identation looks wrong here

Comment (kaikreuzer) [inline launch/app/pom.xml]: Could you add it AFTER all the bom dependencies?

File: launch/app/pom.xml [hunk 2]
```diff
@@ -32,6 +32,11 @@
...
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.zwave</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (kaikreuzer) [inline launch/app/pom.xml]: Not sure if we should also add the runtime scope as for the others?

**Comment by cdjackson** [comment]
Updated

File: launch/app/pom.xml [hunk 3]
```diff
@@ -32,14 +32,22 @@
...
       <version>${project.version}</version>
       <type>pom</type>
       <scope>runtime</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.zwave</artifactId>
+      <version>${project.version}</version>
+      <type>pom</type>
```
Comment (kaikreuzer) [inline launch/app/pom.xml]: This artifact is not a pom - so this line should not be added.

**Pull request merged**